packages/node-sdk/src/client.ts

```diff
@@ -910,6 +910,9 @@ export class BucketClient {
 
         return isEnabled;
       },
+      get config() {
+        return undefined;
+      },
```

Comment:
looks like nodeSDK changes are not complete. Should we do this in a separate PR?